File: Frontend/Webshop/src/app/Dtos/ProductInCartRequestDtos.ts
import {ProductInCart} from '../models/productInCart.model'

export class ProductInCartRequestDtos {
    
    username : string;
    size:string;
    amount: number;
    productName: string;
    productid: number;

    constructor() {

     }
     CopyFromProductInCart(productincart: ProductInCart )
     {
         this.size=productincart.size;
         this.amount=productincart.amount;
         this.productName=productincart.product.productName;
         this.productid=productincart.product.id;
     }
    
  }
  

File: WebshopBackend/WebshopBackend/Dtos/ProductInCartRequestDtos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebshopBackend.Dtos
{
    public class ProductInCartRequestDtos
    {
        public string username { get; set; }

        public string size { get; set; }
        public int amount { get; set; }

        public string productname { get; set; }
        public long productid { get; set; }
    }
}
